tests/unit/transformers/test_numerical.py

```diff
@@ -176,7 +247,7 @@ def test_fit_rounding_auto_max_inf(self):
         transformer.fit(data)
 
         # Asserts
-        assert transformer._rounding_digits is -3
+        assert transformer._rounding_digits == -3
```

Comment:
I'm surprised this didn't throw a lint error

Reply:
Yeah, me too. Because it actually complains now!